ISSUER_PROTOCOL.md

```diff
@@ -18,22 +18,29 @@ A Trust Token issuer should have an endpoint at a publicly accessible secure URL
 ```
 Key commitment result
 {
-  "protocol_version": <protocol version, TrustTokenV3PMB or TrustTokenV3VOPRF for this>,
-  "id": <key commitment identifier, as a monotonically increasing integer>
-  "batchsize": <batch size>,
-  "srrkey": <base-64 encoded SRRVerificationKey, in 32-byte RFC8032 encoding>,
-  <keyID>: { "Y": <base64-encoded TrustTokenPublicKey>,
-                     "expiry": <key expiry, encoded as a string representation of
-                                an integer timestamp in microseconds since the Unix
-                                epoch> },
-  <keyID>: { "Y": <base64-encoded public key>,
-                     "expiry": <key expiry, encoded as a string representation of
-                                an integer timestamp in microseconds since the Unix
-                                epoch> }, ...
+  <protocol_version>: {
+    "protocol_version": <protocol version, TrustTokenV3PMB or TrustTokenV3VOPRF for this>,
+    "id": <key commitment identifier, as a monotonically increasing integer>
+    "batchsize": <batch size>,
+    "srrkey": <base-64 encoded SRRVerificationKey, in 32-byte RFC8032 encoding>,
+    "keys": {
+      <keyID>: { "Y": <base64-encoded TrustTokenPublicKey>,
+                         "expiry": <key expiry, encoded as a string representation of
+                                    an integer timestamp in microseconds since the Unix
+                                    epoch> },
+      <keyID>: { "Y": <base64-encoded public key>,
+                         "expiry": <key expiry, encoded as a string representation of
+                                    an integer timestamp in microseconds since the Unix
+                                    epoch> }, ...
+    }
+  },
+  ...
 }
 ```
 
-The `id` field is used as a unique identifier to identify this key commitment and to allow comparing the freshness of key commitments (larger values indicate a newer key commitment).
+The commitment is a dictionary keyed by protocol version where each value is a self-contained key commitment for that version. To support compatibility with prior
```

Comment:
(no action needed) "The commitment is a dictionary keyed by protocol version": it's not a big issue, but, later on, it might make sense for disambiguation's sake to define a new term for the top-level structure.